azure-keyvault/azure/keyvault/aio/secrets/_client.py

```diff
@@ -61,21 +72,31 @@ def __init__(self, vault_url, credentials, config=None, api_version=None, **kwar
 
     @property
     def vault_url(self):
+         # type: () -> str
```

Comment:
We can use Python 3.5 type hint syntax instead of the comment syntax (check out `list_secrets` for an example).